extensions/wikia/AffiliateService/js/templates.mustache.js

```diff
@@ -0,0 +1,7 @@
+define( 'ext.wikia.AffiliateService.templates', [], function() { 
+    'use strict'; 
+    
+    return {
+        "AffiliateService_unit" : '<div style="display: flex; background: red; width: 100%; height: 100px"><img style="height: 100%;" src="{{image}}"><p>{{heading}}</p><button>{{buttonText}}</button><img src="{{logo}}" /></div>',
```

Comment:
Can we use an external file for the HTML and external CSS?

Reply:
We can and we should

Reply:
Using an external `.scss` file in CAKE-5359